src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/PtvServiceChannelProvider.java

```diff
@@ -72,6 +118,60 @@ public ServiceLocationServiceChannel findServiceLocationServiceChannel(ServiceLo
   }
   
   @Override
+  public IntegrationResponse<ServiceLocationServiceChannel> updateServiceLocationServiceChannel(ServiceLocationServiceChannelId serviceLocationChannelId, ServiceLocationServiceChannel serviceLocationServiceChannel) {
+    ServiceLocationServiceChannelId ptvServiceLocationServiceChannelId = idController.translateServiceLocationServiceChannelId(serviceLocationChannelId, PtvConsts.IDENTIFIER_NAME);
+    if (ptvServiceLocationServiceChannelId != null) {
+      Map<String, Object> channelData = ptvServiceChannelResolver.loadServiceChannelData(ptvServiceLocationServiceChannelId.getId());
+      ServiceChannelType serviceChannelType = ptvServiceChannelResolver.resolveServiceChannelType(channelData);
+      if (serviceChannelType != ServiceChannelType.SERVICE_LOCATION) {
+        logger.log(Level.SEVERE, () -> String.format("Attempted to update ptv %s service channel %s into SERVICE_LOCATION", serviceChannelType, ptvServiceLocationServiceChannelId.getId()));
+        return null;
+      }
+      
+      byte[] serializeChannelData = ptvServiceChannelResolver.serializeChannelData(channelData);
+      V7VmOpenApiServiceLocationChannel ptvServiceLocationServiceChannel = ptvServiceChannelResolver.unserializeServiceLocationChannel(serializeChannelData);
+      V7VmOpenApiServiceLocationChannelInBase ptvServiceLocationServiceChannelIn = ptvOutPtvInTranslator.translateServiceLocationChannel(ptvServiceLocationServiceChannel);
+      
+      if (ptvServiceLocationServiceChannelIn == null) {
+        logger.log(Level.SEVERE, () -> String.format("Failed to translate ptv %s service location service channel", ptvServiceLocationServiceChannelId.getId()));
+        return null;
+      }
+      
+      OrganizationId ptvOrganizationId = ptvIdFactory.createOrganizationId(ptvServiceLocationServiceChannel.getOrganizationId());
+      OrganizationId kuntaApiOrganizationId = idController.translateOrganizationId(ptvOrganizationId, KuntaApiConsts.IDENTIFIER_NAME);
+      if (kuntaApiOrganizationId == null) {
+        logger.log(Level.WARNING, () -> String.format("Failed to translate ptv organization id %s into Kunta API organization id", ptvOrganizationId));
+        return null;
+      }
+      
+      ServiceChannelApi serviceChannelApi = ptvApi.getServiceChannelApi(kuntaApiOrganizationId);
+      
+      ptvServiceLocationServiceChannelIn.setAddresses(kuntaApiPtvTranslator.translateAddressesMovingIn(serviceLocationServiceChannel.getAddresses()));
+//      ptvServiceLocationServiceChannelIn.setAreas(areas);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)